package.json

```diff
@@ -33,6 +33,7 @@
     "pinst": "^3.0.0",
     "prettier": "^2.8.4",
     "prettier-plugin-solidity": "^1.0.0",
+    "serverless-offline": "^12.0.4",
```

Comment:
Is this being used somewhere in the contracts project?

Reply:
to run it locally yes